staging/src/k8s.io/client-go/tools/clientcmd/api/v1/types.go

```diff
@@ -221,6 +221,10 @@ type ExecConfig struct {
 	// to false. Package k8s.io/client-go/tools/auth/exec provides helper methods for
 	// reading this environment variable.
 	ProvideClusterInfo bool `json:"provideClusterInfo"`
+
+	// InteractiveMode determines this plugin's relationship with standard input.
```

Comment:
describe the valid values, the default behaviors for v1alpha1 and v1beta1, and the requirement to specify it for v1

Reply:
9bd7adeb5b3ba24ee2b10eab7c15b33beb009afb